### PR TITLE
Refactoring Login to Match Backend Engineering Standards

### PR DESCRIPTION
**Pull request by hwong0305**

### CODE REVIEW

File: helpers/controllers/authController.test.js [hunk 1]
```diff
@@ -27,23 +27,19 @@ describe('auth controller', () => {
...
-    expect(result).toEqual({
-      success: false,
-      error: 'user does not exist'
-    })
+    return expect(
+      login({}, userArgs, { req: { session: {} } })
+    ).rejects.toThrowError('User does not exist')
   })
 
   test('Login - should return success false if password is invalid', async () => {
```
Comment (coltonehrman) [inline helpers/controllers/authController.test.js:37]: Description wasn't updated with test

File: helpers/controllers/authController.ts [hunk 2]
```diff
@@ -26,35 +26,33 @@ export const login = async (
   arg: Login,
   ctx: { req: Request }
 ) => {
-  const {
-    req: { session }
-  } = ctx
-  const { username, password } = arg
+  try {
```
Comment (coltonehrman) [inline helpers/controllers/authController.ts:29]: Why do you wrap this in a `try...catch` when the catch statement just re-throws the error?

Comment (hwong0305) [reply helpers/controllers/authController.ts:29]: If there's any special case errors from `Sequelize` or `bcrypt` won't propagate properly to `ApolloServer` if it doesn't have a catch.

Comment (songz) [reply helpers/controllers/authController.ts:29]: @coltonehrman not catching will cause nodeJS to `console.warn` that we are not handling error when there are rejections. We probably don't want that if we can help it.

**Pull request merged**